node/src/proxy_server/client_request_payload_factory.rs

```diff
@@ -57,7 +54,7 @@ impl ClientRequestPayloadFactory for ClientRequestPayloadFactoryReal {
                 sequence_number,
                 last_data: ibcd.last_data,
             },
-            target_hostname,
+            target_hostname: target_hostname_opt,
```

Comment:
Can't correct the field name of `ClientRequestPayload_0v1` without migrating to `ClientRequestPayload_0v2`, but I can correct the variable name...